src/com/edlplan/ui/fragment/InGameSettingMenu.kt

```diff
@@ -0,0 +1,566 @@
+            object : OnSeekBarChangeListener {
+                override fun onProgressChanged(
+                    seekBar: SeekBar?,
+                    progress: Int,
+                    fromUser: Boolean
+                ) {
+                    if (fromUser) ModMenu.getInstance().customHP = progress / 10f
+                    customHPText.text = "${progress / 10f}"
+                }
+
+                override fun onStartTrackingTouch(seekBar: SeekBar?) {}
+
+                override fun onStopTrackingTouch(seekBar: SeekBar?) {
+                    ModMenu.getInstance().updateMultiplierText()
+                }
+            }
+        )
+    }
+
+    private fun updateDifficultyAdjustValues() {
+        val track = GlobalManager.getInstance().selectedTrack
+        var visibility = View.VISIBLE
+
+        if (Multiplayer.room != null) {
+            val settings = Multiplayer.room!!.gameplaySettings
+
+            if (!Multiplayer.isRoomHost && (!settings.isFreeMod || !settings.allowForceDifficultyStatistics)) {
+                visibility = View.GONE
+            }
+        }
+
+        forceDifficultyStatisticsSplitter.visibility = visibility
+
+        val customAR = ModMenu.getInstance().customAR
+        customARLayout.visibility = visibility
+        customARToggle.isChecked = customAR != null
+        customARBar.isEnabled = customAR != null
+        customARBar.progress = (((customAR ?: track?.approachRate) ?: 10f) * 10).toInt()
+        customARText.text = "${customARBar.progress / 10f}"
+
+        val customOD = ModMenu.getInstance().customOD
+        customODLayout.visibility = visibility
+        customODToggle.isChecked = customOD != null
+        customODBar.isEnabled = customOD != null
+        customODBar.progress = (((customOD ?: track?.overallDifficulty) ?: 10f) * 10).toInt()
+        customODText.text = "${customODBar.progress / 10f}"
+
+        val customCS = ModMenu.getInstance().customCS
+        customCSLayout.visibility = visibility
+        customCSToggle.isChecked = customCS != null
+        customCSBar.isEnabled = customCS != null
+        customCSBar.progress = (((customCS ?: track?.circleSize) ?: 10f) * 10).toInt()
+        customCSText.text = "${customCSBar.progress / 10f}"
+
+        val customHP = ModMenu.getInstance().customHP
+        customHPLayout.visibility = visibility
+        customHPToggle.isChecked = customHP != null
+        customHPBar.isEnabled = customHP != null
+        customHPBar.progress = (((customHP ?: track?.hpDrain) ?: 10f) * 10).toInt()
+        customHPText.text = "${customHPBar.progress / 10f}"
+
+        ModMenu.getInstance().updateMultiplierText()
+    }
+
+    override fun dismiss() {
+        super.dismiss()
+        uiThread { super.save() }
+        ModMenu.getInstance().hideByFrag()
+    }
+
+    private fun isSettingPanelShow(): Boolean {
+        return abs(findViewById<LinearLayout>(R.id.fullLayout)?.translationY ?: 0f) < 10
```

Comment:
There is a change in behavior here. Previously, if `findViewById` returns `null`, this would return `false`. Now, it returns `true`. Is this intended?

Reply:
Oops, an oversight in my part. I'll change this accordingly.